script.js

```diff
@@ -0,0 +1,578 @@
+        ArrowLeft: '&#8592;',
+      },
+      shift: {
+        Backquote: 'Ё',
+        Digit1: '!',
+        Digit2: '"',
+        Digit3: '№',
+        Digit4: ';',
+        Digit5: '%',
+        Digit6: ':',
+        Digit7: '?',
+        Digit8: '*',
+        Digit9: '(',
+        Digit0: ')',
+        Minus: '_',
+        Equal: '+',
+        BackSpace: 'BackSpace',
+        Tab: 'Tab',
+        KeyQ: 'Й',
+        KeyW: 'Ц',
+        KeyE: 'У',
+        KeyR: 'К',
+        KeyT: 'Е',
+        KeyY: 'Н',
+        KeyU: 'Г',
+        KeyI: 'Ш',
+        KeyO: 'Щ',
+        KeyP: 'З',
+        BracketLeft: 'Х',
+        BracketRight: 'Ъ',
+        Backslash: '/',
+        Delete: 'del',
+        CapsLock: 'CapsLock',
+        KeyA: 'Ф',
+        KeyS: 'Ы',
+        KeyD: 'В',
+        KeyF: 'А',
+        KeyG: 'П',
+        KeyH: 'Р',
+        KeyJ: 'О',
+        KeyK: 'Л',
+        KeyL: 'Д',
+        Semicolon: 'Ж',
+        Quote: 'Э',
+        Enter: 'Enter',
+        ShiftLeft: 'Shift',
+        KeyZ: 'Я',
+        KeyX: 'Ч',
+        KeyC: 'С',
+        KeyV: 'М',
+        KeyB: 'И',
+        KeyN: 'Т',
+        KeyM: 'Ь',
+        Comma: 'Б',
+        Period: 'Ю',
+        Slash: ',',
+        ShiftRight: 'Shift',
+        ControlLeft: 'Ctrl',
+        win: 'win',
+        AltLeft: 'Alt',
+        Space: ' ',
+        AltRight: 'Alt',
+        ControlRight: 'Ctrl',
+        ArrowUp: '&#8593;',
+        ArrowRight: '&#8594;',
+        ArrowDown: '&#8595;',
+        ArrowLeft: '&#8592;',
+      },
+    },
+  },
+
+
+  // method for creating new elements
+  createNewElement(tag, tagClass, tagText) {
+    const elm = document.createElement(tag);
+    elm.className = tagClass;
+    elm.innerHTML = tagText;
+    return elm;
+  },
+
+
+  // method for create KexBox element, arguments:array keys, array function keys
+  createKeyBox(arrKeys, arrFuncKeys) {
+    const keyBox = this.createNewElement('div', 'keyboard', '');
+
+    for (let i = 0; i < arrKeys.length; i += 1) {
+      const keysRow = this.createNewElement('div', 'keys-row', '');
+
+      for (let j = 0; j < arrKeys[i].length; j += 1) {
+        const key = arrKeys[i][j];
+        let classesStr = `key ${key}`.toLowerCase();
+
+        if (arrFuncKeys.includes(key)) classesStr = `${classesStr} key--function`;
+
+        const keyElm = this.createNewElement('button', classesStr, '');
+        keyElm.setAttribute('data-key-code', key);
+        keysRow.append(keyElm);
+      }
+      keyBox.append(keysRow);
+    }
+    return keyBox;
+  },
+
+
+  // method for make keyboard language layout, argument:('en' or 'ru')
+  setLanguageLayout(languageToggle) {
+    const arrKeys = this.elements.keyBox.querySelectorAll('.key');
+    for (let i = 0; i < arrKeys.length; i += 1) {
+      const elem = arrKeys[i];
+      const keySymbol = this.language[languageToggle].lowerCase[elem.dataset.keyCode];
+      elem.innerHTML = keySymbol;
+    }
+    this.inputAria.languageToggle = languageToggle;
```

Comment:
May be <code>inputAr<s>i</s><b>e</b>a</code>?

Reply:
Done